language/move-prover/tests/sources/functional/is_signer.exp

```diff
@@ -0,0 +1,73 @@
+Move prover returns: exiting with boogie verification errors
+error: unknown assertion failed
+
+   ┌── tests/sources/functional/is_signer.move:9:16 ───
+   │
+ 9 │         spec { assert Signer::is_signer(@0x7); } // This is unprovable because it is not true in general.
+   │                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   │
+   =     at tests/sources/functional/is_signer.move:9: f1_incorrect
+
+error: unknown assertion failed
+
+    ┌── tests/sources/functional/is_signer.move:13:16 ───
+    │
+ 13 │         spec { assert Signer::is_signer(@0x7); } // This is unprovable because it is not true in general.
+    │                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    │
+    =     at tests/sources/functional/is_signer.move:12: f2_incorrect
+    =         _account = <redacted>
+    =     at tests/sources/functional/is_signer.move:13: f2_incorrect
+
+error: unknown assertion failed
+
+    ┌── tests/sources/functional/is_signer.move:23:16 ───
+    │
+ 23 │         spec { assert Signer::is_signer(@0x7); } // This is unprovable because it is not true in general.
+    │                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    │
+    =     at tests/sources/functional/is_signer.move:21: f4_incorrect
+    =         account = <redacted>
+    =     at tests/sources/functional/is_signer.move:22: f4_incorrect
+    =         <redacted> = <redacted>
+
+error: precondition does not hold at this call
+
+    ┌── tests/sources/functional/is_signer.move:30:9 ───
```

Comment:
It's not obvious looking at this output that this is error is supposed to happen.  Could you put a comment after the "requires", which will appear in the error message, to indicate that there is supposed to be an error (as you did in some earlier examples)?